skfda/preprocessing/dim_reduction/feature_extraction/_fda_feature_union.py

```diff
@@ -0,0 +1,114 @@
+"""Feature extraction union for dimensionality reduction."""
+from __future__ import annotations
+
+from typing import Union
+
+from numpy import ndarray
+from pandas import DataFrame
+from sklearn.pipeline import FeatureUnion
+
+from ....representation import FData
+
+
+class FdaFeatureUnion(FeatureUnion):  # type: ignore
```

Comment:
Shouldn't the name be `FDAFeatureUnion` instead? FDA is an acronym.

Reply:
You are right, I didn't notice it